### ISSUE TITLE
Error indicators for working code using p5.js

### ISSUE THREAD

**Issue by agreyson** [adobe/brackets#14204]
### Prerequisites

* [x] Can you reproduce the problem with `Debug -> Reload Without Extensions`?
* [x ] Did you perform a cursory search to see if your bug or enhancement is already reported?
* [x ] Did you read the [Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting)?

For more information on how to write a good bug report read [here](https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue)
For more information on how to contribute read [here](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md)

### Description

Ever since installing Brackets 1.12, my p5.js program is littered with error indicators. However, the code runs perfectly. See attached screenshots for both the code and the running program

### Steps to Reproduce

1. Open p5 directory (from [Complete Library download](https://p5js.org/download/)) in Brackets
2. Open sketch.js
3. Save
4. Run in Live Preview

**Expected behavior:** [What you expected to happen]

No error indicators in sketch.js.

**Actual behavior:** [What actually happened]

Error indicators galore.

### Versions

Please include the OS and what version of the OS you're running.
Please include the version of Brackets. You can find it under `Help -> About Brackets` (Windows and Linux) or `Brackets -> About Brackets` (macOS)

MacOS Sierra 10.12.6

Release 1.12 build 1.12.0-17621 (release d3b783b99) 
build timestamp: Thu Feb 01 2018 03:32:03 GMT-0800

![screen shot 2018-04-06 at 4 27 40 pm](https://user-images.githubusercontent.com/5248106/38448438-056c25b4-39b9-11e8-8457-fab57b4a15f3.png)
![screen shot 2018-04-06 at 4 27 27 pm](https://user-images.githubusercontent.com/5248106/38448442-0937decc-39b9-11e8-9007-5137166d63ce.png)

**Comment by JesperSvensson00** [comment]
I have the same problem...

**Comment by agreyson** [comment]
No, unfortunately the problem still exists, and I've heard nothing about it being fixed. If you figure it out, let us know!

**Comment by 84052501** [comment]
I just fix the problem 2 days ago, after 1st time install p5.js with Brackets. For making sure wether a libraries problem, 
1. Put libraries, index.html and schech.js in same directory, didn’t improved! 
   <script src="p5.ps”></script>    <script src="p5.dom.ps”></script> 
 <script src="p5.sound.ps”></script>
<script src="schech.ps”></script>
2. Change to p5.js CDN but didn’t improved! as : 
 <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.3/p5.js"></script> 

So, i don’t think libraries is a problem!

Finally, from the web of “stackoverflow.com” got answer of ‘how to remove error icons in Brackets?’ It suggest modify  the brackets.json file.
"brackets-eslint.gutterMarks": true
Setting it to false.

**Comment by agreyson** [comment]
Interesting. Thank you for investigating. I think you're referring to this stackoverflow thread? https://stackoverflow.com/questions/46949949/how-to-remove-error-icons-in-brackets

Unfortunately, changing it to false turns off ALL error messages, which makes it hard to find true syntax errors. Changing it to { "error": true, "warning" : false } (as described here: https://github.com/brackets-userland/brackets-eslint) causes all the extraneous errors to return. Also, I checked the latest version of Brackets (1.13), but it still has not been fixed.

Anyway, when there was no resolution from Brackets, I finally abandoned it and switched over to using the p5.js web editor (https://editor.p5js.org/), which works great. I highly recommend it!

**Comment by 84052501** [comment]
![螢幕快照 2019-04-07 下午1 54 13](https://user-images.githubusercontent.com/49330286/55679365-13b4bd80-593d-11e9-95f0-fef154061f52.png)
don't know if you can see the attach file? regarding error message after turn "brackets-eslint.gutterMarks" in Bracket.json to false : 
1. JSLint has lots errors, please ignore it. 
2. ESLint do show the real error.

**Comment by agreyson** [comment]
Weirdly, if there's just the syntax error, then the console shows just that error. When I fix the syntax error, the console shows all the extraneous errors. This is after I set
"brackets-eslint.gutterMarks": false,

<img width="616" alt="Screen Shot 2019-04-06 at 11 16 40 PM" src="https://user-images.githubusercontent.com/5248106/55679549-6604c880-58c2-11e9-838c-34f5aed4beae.png">

**Comment by JesperSvensson00** [comment]
I fixed it by just deactivate ESLint. Obviously this will turn off the good about it.
